tests/reftests/env.test

```diff
@@ -651,3 +651,29 @@ OPAM_PACKAGE_NAME=another-package
 OPAM_PACKAGE_VERSION=another-version
 OPAM_SWITCH_PREFIX=${BASEDIR}/OPAM/bd
 PATH=${BASEDIR}/OPAM/bd/bin:another-path
+### : The effect of env updates containing empty strings on variables.sh
```

Comment:
I think that this should be in its own test file. It is indeed environment handling, but more specifically variables.sh environment variable handling, we can begin a test that contains this test in order to update it with other tests specific to variables.sh.